composer.json

```diff
@@ -19,18 +19,18 @@
     "require-dev": {
         "composer/installers": "~1.11",
         "drupal/config_devel": "~1.2",
-        "drupal/core-composer-scaffold": "^9.2",
-        "drupal/core-dev": "^9.2",
+        "drupal/core-composer-scaffold": "^9.3",
+        "drupal/core-dev": "^9.3",
         "drupal/drupal-extension": "~4.0",
         "drush/drush": "^10.3",
         "drupal/honeypot": "^2.0.2",
-        "egulias/email-validator": "^2.1.22 || ^3.0",
+        "fabpot/goutte": "~3.3",
         "openeuropa/code-review": "^1.6 || ^2.0",
         "openeuropa/task-runner-drupal-project-symlink": "^1.0.0-beta5",
         "phpspec/prophecy-phpunit": "^1 || ^2"
     },
     "_readme": [
-        "Explicit requirement for egulias/email-validator due to https://www.drupal.org/project/drupal/issues/3061074#comment-14300579. It can be removed when Drupal core 9.2 support is droppped.",
+        "Explicit minimum version requirement for fabpot/goutte due to lower version doesn't work with Drupal 9.3. Behat tests aren't used but Drone runs them.",
```

Comment:
If we need `fabpot/goutte` only for Behat and there is no Behat scenario other than the example one, please just remove the Behat configuration entirely along with this component.